distribution/ltp/lite/configs/RHELKT1LITE.20180926

```diff
@@ -619,9 +647,10 @@ memset01 memset01
 memcmp01 memcmp01
 memcpy01 memcpy01
 
-migrate_pages01 migrate_pages01
```

Comment:
Looks mostly good, thanks @rasibley ! I have just a tiny request - this test and the comment below seem to be exchanged, so it looks like the comment is related to migrate_pages01 and not 02.

Can you also squash all the commits into one while you're at it? I should be able to do it during the merge if not